operators/pkg/apis/common/v1alpha1/common.go

```diff
@@ -59,11 +61,14 @@ type SubjectAlternativeName struct {
 
 // HTTPService contains defaults for a HTTP service.
 type HTTPService struct {
-	// Metadata is metadata for the HTTP Service.
-	Metadata HTTPServiceObjectMeta `json:"metadata,omitempty"`
+	// ObjectMeta is metadata for the service.
+	// The name and namespace provided here is managed by ECK and will be ignored.
+	// +optional
+	ObjectMeta metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// Spec contains user-provided settings for the HTTP Service.
-	Spec HTTPServiceSpec `json:"spec,omitempty"`
+	// Spec defines the behavior of the service.
+	// +optional
+	Spec v1.ServiceSpec `json:"spec,omitempty"`
 }
 
 // HTTPServiceObjectMeta is metadata for HTTP Service.
```

Comment:
Do we still use HTTPServiceObjectMeta & HTTPServiceSpec?

Reply:
Ah no, I will remove :)